spark/pom.xml

```diff
@@ -198,7 +197,13 @@
             <properties>
                 <spark.version>2.3.0</spark.version>
                 <protobuf.version>2.5.0</protobuf.version>
-                <spark.py4j.version>0.10.6</spark.py4j.version>
+                <py4j.version>0.10.6</py4j.version>
+                <spark.src.download.url>
+                    http://www-us.apache.org/dist/spark/${spark.archive}/${spark.archive}.tgz
+                </spark.src.download.url>
+                <spark.bin.download.url>
+                    http://www-us.apache.org/dist/spark/${spark.archive}/${spark.archive}-bin-without-hadoop.tgz
+                </spark.bin.download.url>
```

Comment:
this is a pre-existing problem but we really need to change to use the asf mirrors instead - we can't release with this...

Reply:
Right, this is tracked. But I afraid it could not be fixed in 0.8. 
@woowahan-jaehoon Zeppelin 0.8 can still run spark 2.3 without this PR. The spark-2.3 profile only affect that it would make spark 2.3 as the embedded version of spark. As @felixcheung  mentioned, we can make this change after the download url is fixed

Reply:
So, how did you solved download issue?
Just using CDN?